release-notes/docs/3.10.0.md

```diff
@@ -0,0 +1,152 @@
+This feature can have a multitude of applications, from providing a way to access an arbitrary contact's phone number when resolving an SMS recipient to linking a delivery report to a child person document (`linked_docs` can also be used in `contact-summary`).   
+//todo documentation link
+
+#### Privacy Policies
+
+App builders can now configure privacy policies for every language. When a privacy policy is configured for a language, users who load the app in this language are prompted to accept the policy, and are blocked from using the app until they have accepted. Privacy policies can be configured using `medic-conf` or from a new App Management page.    
+See [the documentation](https://docs.communityhealthtoolkit.org/apps/guides/security/privacy-policy/) for more information.  
+![privacy policies](./images/3.10-privacy-policies.png)
+
+#### More granular replication depth configuration for offline users
+
+This release adds a new `replication_depth` configuration option that limits the depth of replicated reports. This allows app builders to configure roles that replicate deeper contacts, can report against these contacts, but will not download reports about these contacts that have been created by other users.  
+See [the documentation](https://docs.communityhealthtoolkit.org/apps/guides/performance/replication/#report-depth) for more information.
+
+#### Improved geolocation data
+
+3.10.0 improves the way geolocation information is obtained, from requesting an instant position to allowing up to 30 seconds to get a position lock. Additionally, the `geolocation` property is now updated every time a report is edited, and the all previous geolocation entries are saved in a log within the same report.  
+This change also supports the latest updates in `medic-android`, where Location permission is requested at runtime, when editing/creating a report, rather than demanded when starting the app for the first time.   
+
+#### Improved performance of processing deleted docs
+
+Because processing document deletes, specifically deleting read docs, is relatively time-consuming, increasing linearly by the number of users that exist on the instance, deletion processing is now split into: 
+- generating tombstones: no changes, tombstones are generated like before, within the Sentinel transition processing feed.
+- deleting read docs and info docs: done within a new Sentinel scheduled task that doesn't block the main transitions processing feed
+
+| Version | Number of users | Number of deleted docs | Time to process |
+| ------- | --------------- | ---------------------- | --------------- |
+| 3.9.0 | 1000 | 500 | 60 minutes | 
+| 3.10.0 | 1000 | 500 | 45 seconds |  
+| 3.10.0 | 1000 | 5000 | 11 minutes |  
```

Comment:
At first glance I thought we'd regressed because 11 minutes > 45 seconds... I think we should drop the third line so we're comparing apples to apples.

Also because the unit changes it looks like a minor improvement (eg: 60 somethings to 45 somethings), so make the units all seconds IMO.

Finally, because it breaks up the big wall of text, and because the improvement is so significant, I think we should convert this to a graph so it's visually obvious that 3.10.0 is 80 times quicker int his use case.

Reply:
> 60 somethings to 45 somethings
lol reading be hard 
I'll make a graph :)